Documentation/deploy-master.md

```diff
@@ -154,6 +160,7 @@ spec:
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --client-ca-file=/etc/kubernetes/ssl/ca.pem
     - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
+    - --runtime-config=extensions/v1beta1=true,extensions/v1beta1/thirdpartyresources=true
```

Comment:
I would err on the side of not explaining. The need to do this should go away with k8s 1.3. @aaronlevy WDYT?